tests/unit/amo/components/TestSearchResult.js

```diff
@@ -144,6 +144,97 @@ describe(__filename, () => {
     expect(root).toHaveClassName('SearchResult--theme');
   });
 
+  it('displays the thumbnail image as the default src for static theme', () => {
+    const headerImageThumb = 'https://addons.cdn.mozilla.net/thumb/1.png';
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      type: ADDON_TYPE_STATIC_THEME,
+      previews: [
+        {
+          id: 1,
+          caption: 'Image 1',
+          image_url: 'https://addons.cdn.mozilla.net/full/1.png',
+          thumbnail_url: 'https://addons.cdn.mozilla.net/thumb/1.png',
+          image_size: [400, 200],
+          thumbnail_size: [200, 100],
+        },
+        {
+          id: 2,
+          caption: 'Image 2',
+          image_url: 'https://addons.cdn.mozilla.net/thumb/1.png',
+          thumbnail_url: headerImageThumb,
+          image_size: [400, 200],
+          thumbnail_size: [200, 100],
```

Comment:
I wasn't sure if I need to put in all this data or just the part that I need..?

Reply:
rereading your comment [here](https://github.com/mozilla/addons-frontend/pull/5225#discussion_r201109747) that's what it seems like but wanted to double check..

Reply:
You only need to override the part the test cares about, but the way `fakeAddon` is set up you do need this. I made a suggestion above about creating a `fakePreview` which you could also use here and then you'd only have to override the one property.